### DISCUSSION TITLE
Adaptive sequencing using single cell barcodes

### DISCUSSION THREAD

**Discussion by skyng22003** [LooseLab/readfish#364] [Q&A]
I was wondering how to implement adaptive sequencing using single cell barcodes rather than genomic regions (ie only continuing the sequencing if a certain barcode is matched), and what are the best practices?

Thanks,
Sky

**Comment by mattloose** [comment]
Hi,

This isn't something we have tried currently and the way barcoding is implemented at present in ReadFish is using the built in ONT barcode demultiplexing. If you have a way of integrating single cell barcodes into ONT demultiplexing it might be possible to do something like this. Alternatively you could implement your own scheme to look at the barcode at the front of a read and make a choice depending on what it matched.

Those are just my immediate thoughts!

**Reply by skyng22003** [reply]
Thank you!

If I use the ONT de-multiplexing method will it just ignore the ONT barcodes and read until it get to the single cell barcode?

For setting the adaptive sequencing criteria example below, I would set the target to [ ] to accept any sequence after the barcode?

If I had like 1K barcodes would this be an issue?

[barcodes.barcode01]
name = "barcode01_targets"
control = false
min_chunks = 0
max_chunks = 2
targets = ["chr1", "chr2"]
single_on = "stop_receiving"
multi_on = "stop_receiving"
single_off = "unblock"
multi_off = "unblock"
no_seq = "proceed"
no_map = "proceed"
